src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageSolutionDetailControlModel.cs

```diff
@@ -181,24 +197,6 @@ private async Task UpdateInstalledVersionsAsync(CancellationToken cancellationTo
             AutoSelectProjects();
         }
 
-        /// <summary>
-        /// This method is called from several methods that are called from properties and LINQ queries
-        /// It is likely not called more than once in an action.
-        /// </summary>
-        private async Task<IPackageReferenceContextInfo> GetInstalledPackageAsync(
```

Comment:
I didn't feel this method was adding any valuable benefit, as only 1 place was using it to filter on packageId.